virtualization/windowscontainers/container-networking/architecture.md

```diff
@@ -55,13 +48,24 @@ The Host Networking Service (HNS) and the Host Compute Service (HCS) work togeth
   - HNS creates NAT and IP pools as required
 
 ### Endpoint Creation
-  - HCS creates network compartment per container endpoint
-  - HNS/HCS places v(m)NIC inside network compartment
+  - HNS creates network namespace per container endpoint
+  - HNS/HCS places v(m)NIC inside network namespace
   - HNS creates (vSwitch) ports
   - HNS assigns IP address, DNS information, routes, etc. (subject to networking mode) endpoint
```

Comment:
Reword to "...assigns IP address, DNS information, routes, etc. (subjet to networking mode) **to the **

Reply:
in other words add, "to the" before endpoint for better grammar